tests/training/test_transformation.py

```diff
@@ -360,35 +369,38 @@ def test_onehot_categorizer():
         ("fklearn_feat__region==SP", [0, 0, 1, 0])
     )))
 
-    # Test without hardcoding NaNs
+    expected_output_test_drop_first_hardcode = pd.DataFrame(OrderedDict((
```

Comment:
Would it make sense to add `drop_first`+`no_hardcode` tests? 
Cases like: `expected_output_train_drop_first_no_hardcode` and `expected_output_test_drop_first_no_hardcode`.

Reply:
It has the four combinations:
Line 381: Without hardcoding NaNs & not dropping first column
Line 392:       with hardcoding NaNs & not dropping first column
Line 403: Without hardcoding NaNs & dropping first column
Line 415:       with hardcoding NaNs & dropping first column

Reply:
Yeah, you're right :) They are there. 
I just would expect this case `expected_output_train_drop_first`, for instance, to be named `expected_output_train_drop_first_no_hardcode`. I think it would be more clear.

Reply:
I had not realized those cases were there because I was searching for `drop_first_no_hardcode`. I thought it made more sense, considering the other cases names. What do you think?